### PR TITLE
Fix iOS home screen icons

### PR DESCRIPTION
**Pull request by samit4me**
Somehow the sizes of the iOS home screen icons were messed up by yours truly [in this commit](https://github.com/react-boilerplate/react-boilerplate/commit/333d1515d024c24f9952f0201c0ba14bcb78c216#commitcomment-20969105). Massive thank you to the awesome @tomazy for finding and raising the issue. I tried to blame a bad merge, but I think the PEBKAC 😳

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -13,10 +13,10 @@
 
     <!-- iOS home screen icons -->
     <meta name="apple-mobile-web-app-title" content="react boilerplate">
-    <link rel="apple-touch-icon" sizes="152x152" href="icon-120x120.png">
+    <link rel="apple-touch-icon" sizes="120x120" href="icon-120x120.png">
     <link rel="apple-touch-icon" sizes="152x152" href="icon-152x152.png">
-    <link rel="apple-touch-icon" sizes="180x180" href="icon-167x167.png">
-    <link rel="apple-touch-icon" sizes="167x167" href="icon-180x180.png">
+    <link rel="apple-touch-icon" sizes="167x167" href="icon-167x167.png">
+    <link rel="apple-touch-icon" sizes="180x180" href="icon-180x180.png">
```
Comment (tomazy) [inline app/index.html]: @samit4me shouldn't these`hrefs` be absolute?

Comment (samit4me) [inline app/index.html]: @tomazy Absolutely 😄, puns aside I agree, just puhed up the changes. Out of interest I tested on iPhone 5s (iOS 9.3.1) and it appears to work with or without relative on both pages so must be some Apple magic going on there.

**Comment by samit4me** [comment]
Looks like AppVeyor cannot install Chrome, it's failing for all builds

File: app/index.html (hunk 1 above)
Comment (tomazy) [inline app/index.html]: I think you would see a difference if your app was not served from the root (`https://example.com/`) but a subpath  (`https://example.com/app/`)

**Comment by kopax** [comment]
Just a curious question. How did you plug the coverage validation for the merge ? 
Can every react-boilerplate project have the same ?

**Comment by samit4me** [comment]
Not sure I understand the question?

**Pull request merged**